### ISSUE TITLE
Aria-current missing MacOS + VoiceOver + Chrome support combination

### ISSUE THREAD

**Issue by attilavago** [accessibilitysupported/a11ysupport.io#131]
For `aria-current` there is a missing combination that does not work, but important enough to mention. On **MacOS, VoiceOver in combination with Chrome** renders the `aria-current` unannounced. Chrome having such a large user-base on MacOS it's important to have it mentioned. 

This Screen Reader + Browser combination seems to be missing in general too.
https://a11ysupport.io/tests/tech__aria__aria_current

**Comment by mfairchild365** [comment]
Hi @attilavago - thank you for the note. The core testing combinations are determined by empirical evidence of which screen reader/browser combinations are actually used by screen reader users. The major source of that information is the [WebAIM screen reader user survey](https://webaim.org/projects/screenreadersurvey8/#browsercombos), which states that VoiceOver is very rarely used in combination with Chrome. Additionally, Chrome has historically not worked well with VoiceOver, although that may be changing.

I'll keep this issue open for now as I try to look for evidence that this is in fact a common combination. The landscape is constantly changing. If you have any evidence, please share it here.

**Comment by attilavago** [comment]
> Hi @attilavago - thank you for the note. The core testing combinations are determined by empirical evidence of which screen reader/browser combinations are actually used by screen reader users. The major source of that information is the [WebAIM screen reader user survey](https://webaim.org/projects/screenreadersurvey8/#browsercombos), which states that VoiceOver is very rarely used in combination with Chrome. Additionally, Chrome has historically not worked well with VoiceOver, although that may be changing.
> 
> I'll keep this issue open for now as I try to look for evidence that this is in fact a common combination. The landscape is constantly changing. If you have any evidence, please share it here.

Thanks so much for responding. 🙏 

The latest data on the user-base (in the millions) that I am working with shows a 4.12%, which I think is high enough to justify adding it to the table (and I am happy to contribute myself if that's something you'd like).
Same goes for ChromeVox on ChromeOS, which represents a nearly 47% user-base. Given the fact that education is moving largely to the online due to the pandemic and new trends, and US states and districts are required to purchase accessible 508, and WCAG 2.0 AA compliant solutions by law, the WebAIM numbers, I'd argue, might already be unreliable from this perspective, though incredibly useful in many other ways.

**Comment by mfairchild365** [comment]
Thank you, and you may be right that the pandemic has been a catalyst for change in the screen reader landscape. 

Can you provide the raw data and collection methodology for the numbers that you cited?
